index.html

```diff
@@ -4,8 +4,9 @@
 
 <head>
     <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500" rel="stylesheet" type="text/css">
-    <link href="./pages/css/reset.css" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+    <link href="https://css.wegotpop.com/1.0.0/index.css" rel="stylesheet" type="text/css">
```

Comment:
We should probably also apply the integrity check to the new sheet as well to avoid problems later.